### ISSUE TITLE
Docs missing for Particle.anti_flag

### ISSUE THREAD

**Issue by HDembinski** [scikit-hep/particle#225]
The anti_flag on Particle has no doc-string, but requires some explanation.

**Comment by HDembinski** [comment]
A docstring for Particle.status would also be nice.

**Comment by henryiii** [comment]
These do have docs, but due to Python limitations, you have to look at the main class, not the attribute. You can't attach docs to a slot in Python < 3.7.

These limitations have been lifted from Python 3.8* by allowing `__slots__` to be a dict, but attrs has not yet added a way to support this. If/when it does.

*: Note, the change was in inspect, changing the *behavior* of inspecting an attribute, and dicts were always valid as slots. Tools could possibly even enable inspection in older Python versions if they implement it manually. But we can always run ReadTheDocs with 3.8, so as long as 3.8 works, that's all that matters.

### Example:

```python
class A:
    __slots__ = {"one": "one doc"}
help(A.one)
```

#### Python 3.7:
```
Help on member descriptor __main__.A.one:

one
```

#### Python 3.8:
```
Help on member descriptor __main__.A.one:

one
    one doc
```

**Comment by HDembinski** [comment]
Thanks for the explanation, which is surprisingly subtle. Good to know it is fixed in 3.8+.

**Comment by eduardo-rodrigues** [comment]
@HDembinski, just to reassure you: this feature also annoys me a lot, and I'm happy that things get better in 3.8! I hope you agree that it would be a shame to rewrite the whole `Particle` class at this stage. One indeed gets a fair amount of doc via `help(Particle)`.

BTW, I have material/data for "all" nuclei privately and will try and commit and discuss with you, cf. your issue raised a while back to extend for nuclei.
